mock-certify-plugin/src/main/java/io/mosip/certify/mock/integration/mocks/MdocGenerator.kt

```diff
@@ -0,0 +1,132 @@
+package io.mosip.certify.mock.integration.mocks
+
+import co.nstant.`in`.cbor.CborBuilder
+import co.nstant.`in`.cbor.CborEncoder
+import co.nstant.`in`.cbor.model.DataItem
+import com.android.identity.credential.NameSpacedData
+import com.android.identity.internal.Util
+import com.android.identity.mdoc.mso.MobileSecurityObjectGenerator
+import com.android.identity.mdoc.util.MdocUtil
+import com.android.identity.util.Timestamp
+import io.mosip.certify.util.CBORConverter
+import io.mosip.certify.util.JwkToKeyConverter
+import io.mosip.certify.util.KeyPairAndCertificate
+import io.mosip.certify.util.PKCS12Reader
+import io.mosip.certify.util.UUIDGenerator
+import java.io.ByteArrayOutputStream
+import java.time.Instant
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import java.time.temporal.ChronoUnit
+import java.util.*
+
+
+class MdocGenerator {
```

Comment:
Do we really need this in kotlin code, not possible to keep this also in java ?

Reply:
@KiruthikaJeyashankar please check this suggestion

Reply:
Now tried modifying to Java code and unit tests are passing. This can be modified to Java to maintain consistency

Reply:
this is great, so then we can remove current file changes and do all the restructuring as part of Inji mobile release or we are good to include in the current Inji certify release ?